<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>KisWdgStopGradientEditor</class>
 <widget class="QWidget" name="KisWdgStopGradientEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>291</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1,0">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameedit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="buttonsContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="selectPreviousStopButton"/>
        </item>
        <item>
         <widget class="QToolButton" name="selectNextStopButton"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="selectedStopLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deleteStopButton"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="flipStopsButton"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="sortByValueButton"/>
        </item>
        <item>
         <widget class="QToolButton" name="sortByHueButton"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QToolButton" name="distributeEvenlyButton"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="KisStopGradientSlider" name="gradientSlider" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="compactModeButtonsContainer" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="compactModeSelectPreviousStopButton"/>
          </item>
          <item>
           <widget class="QToolButton" name="compactModeSelectNextStopButton"/>
          </item>
          <item>
           <widget class="QToolButton" name="compactModeMiscOptionsButton"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stopEditorContainer">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="KisGradientColorEditor" name="stopEditor" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisStopGradientSlider</class>
   <extends>QWidget</extends>
   <header location="global">KisStopGradientSlider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisGradientColorEditor</class>
   <extends>QWidget</extends>
   <header location="global">KisGradientColorEditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameedit</tabstop>
  <tabstop>selectPreviousStopButton</tabstop>
  <tabstop>selectNextStopButton</tabstop>
  <tabstop>deleteStopButton</tabstop>
  <tabstop>flipStopsButton</tabstop>
  <tabstop>sortByValueButton</tabstop>
  <tabstop>sortByHueButton</tabstop>
  <tabstop>distributeEvenlyButton</tabstop>
  <tabstop>compactModeMiscOptionsButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
